wifiphisher/pywifiphisher.py

```diff
@@ -766,6 +779,8 @@ def run():
     network_manager = interfaces.NetworkManager(args.jamminginterface,
                                                 args.apinterface)
 
+    mac_matcher = macmatcher.MACMatcher()
```

Comment:
Now this is going to be a relatively large object since it will load all the contents of `nmap-mac-prefixes` and given that it is globally declared, it will only be garbage collected when the script terminates. Given that `wifiphisher` is a script that may run for long time, I believe it makes sense to unbind these from memory. I would create an `unbind` method in the `MACMatcher` class for this that would either `del` `mac_to_vendor` or set it to `None`. What do you say?